code_comment/lib.py

```diff
@@ -116,13 +170,14 @@ def is_single_line_comment_multiline_notation(text):
                 not is_currently_multi_line_comment()
                 and text.startswith(mlc_header)
                 and text.endswith(mlc_footer)
+                and len(text) >= 6
```

Comment:
question:

why is this 6? I'm guessing for Python, this is because `"""` X 2? Is that true for all other languages too?

Either case, this 'hardcoded' value seems a little hacky to me, still.

A better way would be to use what we defined in `MULTILINE_COMMENT` constant in each language class

```suggestion
                and len(text) >= len(mlc_header + mlc_footer)
```